packages/adapters/ethers5/src/client.ts

```diff
@@ -829,7 +829,6 @@ export class Ethers5Adapter {
       return
     }
 
-    this.appKit?.setIsConnected(true, this.chainNamespace)
```

Comment:
Getting rid of `setIsConnected` here since we'll use `ChainController.state.activeCaipAddress` to see if a user is connected